### PR TITLE
Create eip155-1515.json

### PR DESCRIPTION
**Pull request by 0xli**
Add Beagle Messaging Chain which has run for more than 3 years

### CODE REVIEW

File: _data/chains/eip155-1515.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "symbol": "BG",
+    "decimals": 18
+  },
+  "infoURL": "https://beagle.chat/",
+  "shortName": "beagle",
+  "chainId": 1515,
+  "networkId": 1515,
+  "explorers": [{
+    "name": "beagle chain explorer",
+    "url": "https://eth.beagle.chat",
```
Comment (ligi) [inline _data/chains/eip155-1515.json:19]: I cannot reach this site

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by 0xli** [comment]
Can you access the RPC and infoURL site?
It should work.
We are still working on the faucet and explore site. Can I delete them and
add back later?

Wei Li

ligi ***@***.***> 于 2022年8月17日周三 18:35写道：

File: _data/chains/eip155-1515.json (hunk 1 above)
Comment (0xli) [reply _data/chains/eip155-1515.json:19]: it works now.

**Comment by ligi** [comment]
yes - you can delete and add the details later - but I do not want to merge not working stuff

**Comment by 0xli** [comment]
> yes - you can delete and add the details later - but I do not want to merge not working stuff

it works now, please check https://eth.beagle.chat

**Comment by 0xli** [comment]
The explore is up but it is not stable yet, I am going to remove it
tomorrow. What's the minimal required information needed in order to be
listed on your esteemed site?

ligi ***@***.***> 于 2022年8月18日周四 17:16写道：

**Comment by 0xli** [comment]
I am going to delete some information first

**Pull request closed without merging**